daemon/gluster-blockd.c

```diff
@@ -112,11 +120,11 @@ glusterBlockCliThreadProc (void *vargp)
 
   if (!svc_register(transp, GLUSTER_BLOCK_CLI, GLUSTER_BLOCK_CLI_VERS,
                     gluster_block_cli_1, IPPROTO_IP)) {
-		LOG("mgmt", GB_LOG_ERROR,
+                LOG("mgmt", GB_LOG_ERROR,
         "unable to register (GLUSTER_BLOCK_CLI, GLUSTER_BLOCK_CLI_VERS: %s)",
         strerror (errno));
     goto out;
-	}
+        }
```

Comment:
There is nothing changed except adding some spaces.

Reply:
Done